packages/reactive_ble_mobile/pubspec.lock

```diff
@@ -350,9 +350,9 @@ packages:
   reactive_ble_platform_interface:
     dependency: "direct main"
     description:
-      name: reactive_ble_platform_interface
-      url: "https://pub.dartlang.org"
-    source: hosted
+      path: "../reactive_ble_platform_interface"
```

Comment:
same here the relative path is needed for development but should not be integrated